src/services/storage-service.js

```diff
@@ -216,6 +216,63 @@ export default class StorageService {
     }
   }
 
+  async getRoomMediaOverview({ user }) {
+    const storagePlan = user.storage.planId ? await this.getStoragePlanById(user.storage.planId) : null;
+    const rooms = await this.roomStore.getRoomsByOwnerId(user._id);
+    const roomsAndObjects = await Promise.all(rooms.map(async room => {
+      const objects = await this.getObjects({ parentPath: getRoomMediaRoomPath(room._id) });
+      return { room, objects, usedBytes: objects.reduce((accu, obj) => accu + obj.size, 0) };
+    }));
+
+    const totalUsedBytes = roomsAndObjects.reduce((accu, { usedBytes }) => accu + usedBytes, 0);
+
+    return {
+      storagePlan: storagePlan || null,
+      usedBytes: totalUsedBytes,
+      unusedBytes: Math.max(0, (storagePlan?.maxBytes || 0) - totalUsedBytes),
+      rooms: roomsAndObjects.map(({ room, objects, usedBytes }) => ({
```

Comment:
Maybe I'd call this `storagePerRoom`/`roomStorages` or something of sorts, other than `rooms`. I find it risky to reuse entity names because we might find ourselves somewhere else supposing that items in this array have `_id`, `_name` etc.

Reply:
Meantime I did notice that there would be repetition in usage (`storage.roomStorages`), in the end just like for `storage.storagePlan`, but I do think it would be much clearer in cases like

```
 const newRoomOptions = storage.rooms
      .map(room => ({ value: room.roomId, label: room.roomName }))
```

where if I look at the code I really might expect something else inside a `room` variable and misuse it 🤔 

Does this make sense?

Reply:
Yes, totally. I had a hard time finding a name for this. I first went with `roomDetails`, but later realized I need something that works in singular and plural (as one response contains all rooms of a user, the other one just a single room, but the data structure is identical).

I like `roomStorage(s)` the most of your suggestions. Maybe it would make sense to go for two different names like
* `roomDetails` for the (GET/POST/DELETE) `/api/v1/storage/room-media/:roomId/:name` endpoints
* `roomOverview` for the (GET) `/api/v1/storage/room-media-overview` endpoint (with the list of `roomDetails` for each room)

What do you think?

Reply:
So, as we talked, weI'll go for `roomStorage` and `roomStorageList` :+1:

Reply:
Yes, let's 👍

Reply:
Done. I also renamed the props in the storage tab to reflect the entity it is supposed to be set to: `roomMediaOverview` and added proper prop type validation (it was set to generic `object` before), you can check the last commit: https://github.com/educandu/educandu/pull/932/commits/a03a71a4434378b144120388cb980a91206426ff

Reply:
Perfect, it's much clearer!